homeassistant/components/media_player/const.py

```diff
@@ -29,6 +29,29 @@
 
 DOMAIN = "media_player"
 
+MEDIA_CLASS_ALBUM = "album"
+MEDIA_CLASS_APP = "app"
+MEDIA_CLASS_APPS = "apps"
+MEDIA_CLASS_ARTIST = "artist"
+MEDIA_CLASS_CHANNEL = "channel"
+MEDIA_CLASS_CHANNELS = "channels"
+MEDIA_CLASS_COMPOSER = "composer"
+MEDIA_CLASS_CONTRIBUTING_ARTIST = "contributing_artist"
+MEDIA_CLASS_DIRECTORY = "directory"
```

Comment:
I'm using `"directory` as a common id for items that contain other items but can't be specified more closely. Other names that are used today as media content type:
- `"library"`
- `"listing"`